create_box/main.py

```diff
@@ -1,5 +1,13 @@
 def create_box(height, width, character):
-    pass
+    height = int(height)
```

Comment:
No need to pass height and width to int(). Check the tests, they are receiving integers as arguments.